client/datasync-client/README.md

```diff
@@ -1,21 +1,19 @@
 # RainCatcher DataSync
 
-Feedhenry Sync typescript client.
-Data synchonization javascript client. Library can be used for offline storage of mobile applications data.
-
-Raincatcher DataSync is based on [sync.js client](https://github.com/feedhenry/fh-sync-js)
-For advanced usage see [FeedHenry Sync documentation](https://github.com/feedhenry/fh-sync/tree/master/docs).
+Raincatcher DataSync is based on the[Feedhenry sync.js client](https://github.com/feedhenry/fh-sync-js)
+For advanced usage of the Raincatcher DataSync module, see[FeedHenry Sync] (https://github.com/feedhenry/fh-sync/tree/master/docs) documentation.
 
 ## Overview
 
-Sync mobile data synchronization framework provides the following key functionality:
+Raincatcher DataSync provides the following functionality:
 
+- Allows synchronisation of WFM workorder data between the RainCatcher mobile client and the RainCatcher cloud server
 - Allows mobile apps to use and update data offline (local cache)
```

Comment:
maybe "Allows synchronisation of a data" as it's not limited to workorder data only AFAIK?